src/Blaster.WebApi/Features/Shared/components/Shared.js

```diff
@@ -16,9 +16,25 @@ new Vue({
         showIEBanner: function () {
             return isIE();
         }
+    },
+    methods: {
+        toggleActive: function () {
+            this.active = this.active ? false : true;
```

Comment:
`this.active = !this.active;`

Reply:
https://github.com/dfds/blaster/pull/135/commits/409e5dbe22896734c8545fce24ceccda66d14fa1 Gotcha ✔